docs/user/02-configuration.md

```diff
@@ -73,6 +73,22 @@ Use the following sample CRs as guidance. Each can be applied immediately when y
 | **publisherService**                                 | string     |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
 | **specHash** (required)                              | integer    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
-| **state** (required)                                 | string     | Can have one of the following values: Ready, Error, Processing, Warning. Ready state is set when all the resources are deployed successfully and backend is connected. It gets Warning state in case backend is not specified and NATS module is not installed or EventMesh secret is missing in the cluster. Error state is set when there is an error. Processing state is set if recources are being created or changed. |
+| **state** (required)                                 | string     | Can have one of the following values: `Ready`, `Error`, `Processing`, `Warning`. `Ready` state is set when all the resources are deployed successfully and backend is connected. It gets `Warning` state in case backend is not specified, NATS module is not installed or EventMesh secret is missing in the cluster. `Error` state is set when there is an error. `Processing` state is set if recources are being created or changed. See the next section for more details. |
 
 <!-- TABLE-END -->
+
+<h3>More Details about Eventing CR state</h3>
+The following table provides more details on the overall state of the Eventing CR:
+- The <b>State</b> column has the overall state of the Eventing CR.
+- The <b>Backend Config</b> column is part of the eventing CR that is either available or mistakenly not specified by a user.
+- The <b>Backend State</b> column describes the state of the NATS backend or EventMesh Secret existence. 
+
+Warnings indicate that user action is required, that is, the user must install the NATS module or create an EventMesh Secret.
+
+| Backend Config | Backend State         | State |
```

Comment:
the idea of this ticket was to clearly document which combination of settings will cause what sort of state
eg:

----
EV-configuration  backend: NATS
NATS cluster: not available
resulting EV-STATE: ? (no events will be accepted / dispatched)

-----
EV backend: NATS
NATSCluster: available (even if the NATS-module is currently being deleted)
Eventing: READY (events will still be accepted / dispatched)

....

Reply:
Second case you mentioned is still under development. I either document and wait until Marco is finished. Or, Marco will document it here. What do do think?